datasets/us/ky/med_exclusions/us_ky_med_exclusions.yaml

```
@@ -0,0 +1,69 @@
+title: US Kentucky State Medicaid Terminations
+entry_point: crawler.py
+prefix: us-medky
+coverage:
+  frequency: daily
+  start: "2024-11-08"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers terminated from Kentucky State's Medicaid program.
+description: |
+  This dataset contains individuals and entites that have been terminated
+  from Kentucky's Medicaid Program. The following text explains the 
+  conditions and implications of such terminations or exclusions,
+  including the restrictions on reimbursement, re-enrollment criteria,
+  and responsibilities for Medicaid providers in screening employees and
+  contractors:
+
+   > An individual or entity terminated or excluded from participating in
+   > the Kentucky Medicaid program will not be reimbursed for services
+   > provided in any capacity or in any category under the Kentucky Medicaid
+   > program. The termination or exclusion remains in effect until the
+   > provider is removed from this list. Where applicable and when determined,
+   > an end date is provided. It is the provider's responsibility to notify
+   > Kentucky Medicaid if the circumstances change which led to the termination
+   > and/or exclusion. A provider terminated by the Department for Medicaid
+   > Services who meets the requirements of 907 KAR 1:671 and 907 KAR 1:672
+   > may apply for re-enrollment in the Kentucky Medicaid program. Any provider
+   > participating or applying to participate in the Kentucky Medicaid program
+   > must search the list of excluded individuals and entities and the System
+   > for Award Management​ on a monthly basis to determine if any existing
+   > employee or contractor has been terminated or excluded from participation
+   > in the Kentucky Medicaid program or has been nationally excluded from
+   > Medicare or Medicaid. Also, any provider participating or applying to
+   > participate in the Kentucky Medicaid program must search all lists prior
+   > to hiring staff to ensure that any potential employee or contractor has
+   > not been terminated and/or excluded from participating in the Medicare
+   > or Medicaid program. Please note that the Kentucky Medicaid Program only
+   > lists providers terminated or excluded from the Kentucky Medicaid program.
+   > HHS/OIG-excluded individuals and entities are not allowed to receive
+   >reimbursement for providing Kentucky Medicaid services in any capacity,
+   > even if they are not on the Kentucky Medicaid list.
+publisher:
+  name: Kentucky Department of Medicaid Services
+  description: >
+    > Kentucky Medicaid is a state and federal program authorized by Title XIX
```

remember to use the block style multinline indicator when you want markdown block semantics in the publisher description https://yaml-multiline.info/

```suggestion
  description: |
    > Kentucky Medicaid is a state and federal program authorized by Title XIX
```